pkg/controllermanager/controller/add.go

```diff
@@ -66,7 +74,14 @@ func AddControllersToManager(mgr manager.Manager, cfg *config.ControllerManagerC
 		return fmt.Errorf("failed adding Quota controller: %w", err)
 	}
 
-	if evenControllerConfig := cfg.Controllers.Event; evenControllerConfig != nil {
+	if err := (&certificatesigningrequest.Reconciler{
+		CertificatesClient: kubernetesClient,
```

Comment:
Let's only pass the CSR client here.

```suggestion
		CertificatesClient: kubernetesClient.CertificatesV1().CertificateSigningRequests(),
```